### PR TITLE
Apply shared checkstyles to all samples.

### PR DESCRIPTION
**Pull request by tswast**
We use git submodules, so to check out all the code for
`java-docs-samples`, we need to clone with the `--recursive` flag.

```
git clone --recursive git@github.com:GoogleCloudPlatform/java-docs-samples.git
```

To check out the directory in an already-cloned repo, run:

```
git submodule update --init --recursive
```

Depends on https://github.com/GoogleCloudPlatform/java-docs-samples/pull/114.

### CODE REVIEW

**Comment by tswast** [comment]
This should complete the `unittests` item in https://github.com/GoogleCloudPlatform/java-docs-samples/issues/61.

**Comment by tswast** [comment]
Build will likely fail due to https://github.com/GoogleCloudPlatform/java-docs-samples/issues/118. I will send a PR to fix that first.

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "java-repo-tools"]
```
Comment (lesv) [inline .gitmodules]: I'm sure this is different than the last for a reason?

**Comment by lesv** [comment]
LGTM

**Pull request merged**